go/ldflags-lrt.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux || netbsd
```

Comment:
How about Free and Open BSD?

Just a side note, but on newer glibc (around 2.17 IIRC), explictly linking against rt is no longer required.

Has this been run on buildbot?